test/controllers/unattended_controller_test.rb

```diff
@@ -666,5 +666,27 @@ class UnattendedControllerTest < ActionController::TestCase
         assert_includes @response.body, 'Test build'
       end
     end
+
+    context 'with a host in bootstrap mode' do
+      test 'should render an ipxe error message' do
+        get :host_template, params: { kind: 'iPXE', bootstrap: 1 }, session: set_session_user
+        assert_response :not_found
+        assert_includes @response.body, 'iPXE intermediates'
```

Comment:
This should be `iPXE intermediate` (no `s`) to fix the tests.

Reply:
yep got this